### PR TITLE
test public links with different roles

### PR DESCRIPTION
**Pull request by individual-it**
## Description
test setting roles of public links

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [x] Tests

## Checklist:

- [ ] Code changes
- [ ] Unit tests added
- [ ] Acceptance tests added
- [ ] Documentation ticket raised: <link> 

## Open tasks:

- [ ] ...

### CODE REVIEW

**Comment by individual-it** [comment]
all tests green, but CI failed :thinking: restarting

File: tests/acceptance/features/webUISharingPublic/shareByPublicLink.feature [hunk 1]
```diff
@@ -26,6 +26,40 @@ Feature: Share by public link
...
+      | permissions | <permissions>  |
+      | path        | /simple-folder |
+      | name        | Public link    |
+    And a link named "Public link" should be listed with role "<role>" in the public link list of folder "simple-folder" on the webUI
+    When the public uses the webUI to access the last public link created by user "user1"
+    Then file "lorem.txt" should be listed on the webUI
+    Examples:
+      | role        | permissions                  |
+      | Viewer      | read                         |
+      | Contributor | read, change, create, delete |
```
Comment (PVince81) [inline tests/acceptance/features/webUISharingPublic/shareByPublicLink.feature]: as discussed in the other ticket, this is wrong ?
https://github.com/owncloud/phoenix/issues/1979

Comment (PVince81) [reply tests/acceptance/features/webUISharingPublic/shareByPublicLink.feature]: fix is here https://github.com/owncloud/phoenix/pull/1985. please adjust expectation

**Comment by individual-it** [comment]
test cherry-picked into  #1985

**Pull request closed without merging**